tests/app/views/contexts/test_summary_context.py

```diff
@@ -82,14 +82,33 @@ def test_build_summary_rendering_context(self):
             self.language,
             self.schema,
             self.answer_store,
-            self.progress_store,
             self.list_store,
+            self.progress_store,
```

Comment:
Not sure why the position change

Reply:
They were the wrong way around, now matches the `QuestionnaireSummaryContext` order in the class